registry/handlers/api_test.go

```diff
@@ -1107,7 +1107,7 @@ func testBlobDelete(t *testing.T, env *testEnv, args blobArgs) {
 	ref, _ := reference.WithDigest(imageName, layerDigest)
 	layerURL, err := env.builder.BuildBlobURL(ref)
 	if err != nil {
-		t.Fatalf(err.Error())
+		t.Fatal(err.Error())
```

Comment:
As we're updating these, these can be simplified to;

```go
t.Fatal(err)
```

(same for the other ones)

Reply:
Actually, that's a great shout. I should have realised that. Thanks, will update

Reply:
Yeah, it's one of those cases that either way "works" so may not be worth the code-churn to update existing uses, but as we're touching these lines, might as well do it ☺️